src/lxc/conf.c

```diff
@@ -2741,6 +2741,16 @@ static int instantiate_vlan(struct lxc_handler *handler, struct lxc_netdev *netd
 
 	DEBUG("instantiated vlan '%s', ifindex is '%d'", " vlan1000",
 	      netdev->ifindex);
+	if (netdev->mtu) {
+		int mtu = atoi(netdev->mtu);
+		err = mtu > 64 && mtu < 65536 ? lxc_netdev_set_mtu(peer,mtu) : EINVAL;
+		if(err) {
+			ERROR("failed to set mtu '%s' for %s : %s",
+			      netdev->mtu, peer, strerror(-err));
```

Comment:
Afaict, this will cause `strerror()` to print `Unknown error ...`. E.g. if you compile

```
#include <errno.h>
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int main(int argc, char *argv[])
{
        errno = ENOMEM;
        printf("%s\n", strerror(-errno));
        exit(EXIT_SUCCESS);
}
```
you'd get `Unknown error -12`.